src/components/pages/deduplicate/SelectedDuplicatesOptions.tsx

```diff
@@ -0,0 +1,117 @@
+import { IconButton } from 'components/Container';
+import CloseIcon from 'components/icons/CloseIcon';
+import DeleteIcon from 'components/icons/DeleteIcon';
+import { SetDialogMessage } from 'components/MessageDialog';
+import Navbar from 'components/Navbar';
+import React from 'react';
+import { OverlayTrigger } from 'react-bootstrap';
+import styled from 'styled-components';
+import constants from 'utils/strings/constants';
+
+interface Props {
+    setDialogMessage: SetDialogMessage;
```

Comment:
this complete component is a duplicate of `selectedFileOptions` na, 

I think a better way of doing this would be to have [this part](https://github.com/ente-io/bada-frame/pull/453/files#diff-1411c25bb4dd24de9bd932af0cbdb2bdb5ded3ba3c4bfba771c436c312c99b86R79-R112)  as a component and pass `isDeduplicatePage` to `selectedFileOptions` and conditionally render those options